src/styles/components/_stars.scss

```diff
@@ -0,0 +1,25 @@
+.stars-list {
+  display: flex;
+}
+
+.stars-item {
+  width: 16px;
+  height: 16px;
+  background: url('../images/star.svg') center no-repeat;
+}
+
+.stars-list--active .stars-item:nth-child(-n + 4) {
+  background-image: url('../images/star-active.svg');
```

Comment:
- It should be the same as in the stars task
- Use nesting in `scss`
- It should have a modifier with a count of active stars, not active. If the modifier is `&--3` you should show 3 active stars, if `--4` - 4 active stars and so on